exercises/pythagorean-triplet/src/Triplet.hs

```diff
@@ -1,7 +1,10 @@
 module Triplet (isPythagorean, mkTriplet, pythagoreanTriplets) where
 
-isPythagorean = error "You need to implement this function."
+isPythagorean :: (Int, Int, Int) -> Bool
```

Comment:
Yeah, I left this exercise for last because I couldn't figure out a way to declare the types without over-specifying it :/

I was completely unaware of https://github.com/exercism/xhaskell/blob/master/exercises/pythagorean-triplet/.meta/DONT-TEST-STUB though

What about get rid of this commit so this exercise can get redesigned as suggested by @rbasso ?